src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person =>
+      person.sex === 'm' && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
+  const sumOfAges
+  = men.reduce((sum, person) => sum + (person.died - person.born), 0);
+
+  return sumOfAges / men.length;
```

Comment:
It would be great to create a function helper, that receives array of people and return their average age

Reply:
Nor resolved. Just create a function helper, that calculates the average value.